### PR TITLE
LPS-96478 Updated the fragment for the links section

### PR DESCRIPTION
**Pull request by eduardoallegrini**

### CODE REVIEW

File: modules/apps/frontend-theme-westeros-bank/frontend-theme-westeros-bank-site-initializer/src/main/resources/com/liferay/frontend/theme/westeros/bank/site/initializer/internal/dependencies/fragments/links.html [hunk 1]
```diff
@@ -1,69 +1,80 @@
-<div class="wb-links">
-	<div class="wb-iconLinks">
-		<div class="container-fluid container-fluid-max-xl">
+<div class="westeros-links westeros-section">
+	<section class="bg-info westeros-bg-info py-5 text-white">
```
Comment (jbalsas) [inline modules/apps/frontend-theme-westeros-bank/frontend-theme-westeros-bank-site-initializer/src/main/resources/com/liferay/frontend/theme/westeros/bank/site/initializer/internal/dependencies/fragments/links.html]: We need to sort the classes here :)

File: modules/apps/frontend-theme-westeros-bank/frontend-theme-westeros-bank-site-initializer/src/main/resources/com/liferay/frontend/theme/westeros/bank/site/initializer/internal/dependencies/fragments/links.html [hunk 2]
```diff
@@ -1,69 +1,80 @@
...
+		<div class="container">
 			<div class="row">
 				<div class="col-12">
-					<h2 class="h6 text-uppercase">
-						<lfr-editable id="linksTitle" type="text">Find Us</lfr-editable>
-					</h2>
+					<lfr-editable id="westeros-links-title" type="rich-text">
+						<h1 class="mb-0 h6">FIND US</h1>
```
Comment (jbalsas) [inline modules/apps/frontend-theme-westeros-bank/frontend-theme-westeros-bank-site-initializer/src/main/resources/com/liferay/frontend/theme/westeros/bank/site/initializer/internal/dependencies/fragments/links.html]: Sort classes here... also, is the uppercase change intentional?

File: modules/apps/frontend-theme-westeros-bank/frontend-theme-westeros-bank-site-initializer/src/main/resources/com/liferay/frontend/theme/westeros/bank/site/initializer/internal/dependencies/fragments/links.html [hunk 3]
```diff
@@ -1,69 +1,80 @@
...
+					</lfr-editable>
 				</div>
 			</div>
 
-			<div class="row" id="footer-modules">
-				<div class="col-sm-6">
-					<div class="icon-phone user-icon user-icon-xl"></div>
+			<div class="row westeros-links-decoration position-relative mb-5">
```
Comment (jbalsas) [inline modules/apps/frontend-theme-westeros-bank/frontend-theme-westeros-bank-site-initializer/src/main/resources/com/liferay/frontend/theme/westeros/bank/site/initializer/internal/dependencies/fragments/links.html]: Sort classes

File: modules/apps/frontend-theme-westeros-bank/frontend-theme-westeros-bank-site-initializer/src/main/resources/com/liferay/frontend/theme/westeros/bank/site/initializer/internal/dependencies/fragments/links.html [hunk 4]
```diff
@@ -1,69 +1,80 @@
...
+				<div class="col-12 col-md-6 d-flex align-items-center mt-5">
```
Comment (jbalsas) [inline modules/apps/frontend-theme-westeros-bank/frontend-theme-westeros-bank-site-initializer/src/main/resources/com/liferay/frontend/theme/westeros/bank/site/initializer/internal/dependencies/fragments/links.html]: sort

File: modules/apps/frontend-theme-westeros-bank/frontend-theme-westeros-bank-site-initializer/src/main/resources/com/liferay/frontend/theme/westeros/bank/site/initializer/internal/dependencies/fragments/links.html (hunk 2 above)
Comment (eduardoallegrini) [reply modules/apps/frontend-theme-westeros-bank/frontend-theme-westeros-bank-site-initializer/src/main/resources/com/liferay/frontend/theme/westeros/bank/site/initializer/internal/dependencies/fragments/links.html]: About the uppercase, yes it is intentional. I didn't want to limit the options in the edit mode, with this setup you can change it manually

**Pull request closed without merging**